xarray/tests/test_backends.py

```diff
@@ -529,7 +529,7 @@ def test_roundtrip_string_encoded_characters(self) -> None:
             assert actual["x"].encoding["_Encoding"] == "ascii"
 
     def test_roundtrip_numpy_datetime_data(self) -> None:
-        times = pd.to_datetime(["2000-01-01", "2000-01-02", "NaT"])
+        times = pd.to_datetime(["2000-01-01", "2000-01-02", "NaT"], unit="ns")
```

Comment:
`pandas.to_datetime` will infer the precision from the input in pandas 3, so we explicitly specify the desired precision now.